src/Shell/Task/BakeTask.php

```diff
@@ -141,9 +141,9 @@ public function callProcess($command)
 
         $error = stream_get_contents($pipes[2]);
         fclose($pipes[2]);
-        proc_close($process);
+        $exit = proc_close($process);
 
-        if ($error) {
+        if ($exit != 0) {
```

Comment:
well, it's academic really. We know it's a number and to me this is clearer:

    if ($exit) {

instead of:

    if ($exit !== 0) {

For me that involves mental shuffling whereas "if exit go here" does not.